source/sass/component/_timeline.scss

```diff
@@ -0,0 +1,148 @@
+$c-timeline-card-text-color: var(--c-timeline-card-text-color, black);
+$c-timeline-card-background-color: var(--c-timeline-card-background-color, white);
+$c-timeline-marker-background-color: var(--c-timeline-marker-background-color, black);
+$c-timeline-marker-border-color: var(--c-timeline-marker-border-color, white);
+$c-timeline-marker-text-color: var(--c-timeline-marker-text-color, white);
+
+.c-timeline {
+    position: relative;
+    margin: 0 auto;
+    padding: 0;
+
+    &::after {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 50%;
+        transform: translateX(-50%);
+
+        width: 1px;
+        background-color: black;
+    }
+}
+
+/* Container around content */
+.c-timeline__event {
+    position: relative;
+    list-style: none;
+    padding: 0;
+
+    .c-timeline__event__card {
+        width: calc(50% - calc(#{$base} * 5));
+        background-color: #{$c-timeline-card-background-color};
+        color: #{$c-timeline-card-text-color};
+    }
+
+    .c-timeline__event__card {
+        &::before {
+            content: ' ';
+            position: absolute;
+            top: 50%;
+            transform: translateY(-50%);
+            right: -10px;
+            height: 0;
+            width: 0;
+            z-index: 1;
+            border: medium solid #{$c-timeline-card-background-color};
```

Comment:
Medium seems like a "loose" value. Not familiar with it, is it possible to set another, more specific value?